global/prometheus-alertmanager-operated/templates/am-config-inhibit-rules-base.yaml

```diff
@@ -52,10 +52,16 @@ spec:
         - matchType: "=~"
           name: severity
           value: 'critical|warning'
+        - matchType: "=~"
+          name: context
+          value: '.+'
```

Comment:
```suggestion
        - matchType: "!="
          name: context
          value: ''
```

Reply:
I'm assuming that means you are confident that `!=` is allowed here. I was going to use that first, but did not find good documentation on which match types are allowed, so I kept to what was already used in the file.

Reply:
Yes, you can double check within the link I shared in Company Slack

Reply:
The Matcher is allowed:

https://github.com/prometheus-operator/prometheus-operator/blob/9092aa95a3acfd4c5caedacd0afffd315cf5a154/pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go#L1039-L1044